apps/desktop/src/renderer/i18n/locales/en.json

```
@@ -2707,6 +2707,12 @@
     "galleryView": "Gallery View",
     "listView": "List View",
     "galleryColumnsLabel": "Skill card columns",
+    "tagFilterTooltip": "Filter by tag",
+    "tagFilterTooltipActive": "Filter by tag ({{count}} active)",
+    "tagFilterSearch": "Search tags",
+    "noTagMatches": "No matching tags",
+    "removeTag": "Remove \"{{tag}}\"",
```

Biome (2.5.8)

[error] The key removeTag was already declared.

(lint/suspicious/noDuplicateObjectKeys)